scripts/trelloOnboardingBuilder.js

```diff
@@ -28,6 +28,15 @@ var app = new Vue({
 
             return boardIdRegexMatches[1];
         },
+        learningPathCardId() {
+            if (! this.learningPathCard) { return ""; }
+
+            const cardIdRegex = /https:\/\/trello\.com\/c\/([^\/]+)\/.*/;
```

Comment:
@AdriAt360 You can remove the regex ending `.*` which does nothing

Reply:
@pp0rtal it's indeed of no use but documentation here
do you think it has an impact to keep it? Unless, I think I will keep it ^^